templates/common/_base/files/kubelet-auto-sizing.yaml

```diff
@@ -46,15 +46,16 @@ contents:
         total_cpu=$(getconf _NPROCESSORS_ONLN)
         recommended_systemreserved_cpu=0
         if (($total_cpu <= 1)); then # 6% of the first core
-            recommended_systemreserved_cpu=$(echo $total_cpu 0.60 | awk '{print $1 * $2}')
+            recommended_systemreserved_cpu=$(echo $total_cpu 0.06 | awk '{print $1 * $2}')
```

Comment:
A typo, we need to get 6% of the CPU. Because of the typo, it was calculating 60%. However, since we never have system in CI or in production with just 1 CPU. This bug never surfaced.